### PR TITLE
i3status-rust: update to 0.31.8.

### PR DESCRIPTION
**Pull request by pablomancera**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I built this PR locally for these architectures:
  - aarch64
  - aarch64 (crossbuild)
  - armv6l (crossbuild)

### CODE REVIEW

**Comment by Bnyro** [comment]
(supersedes #44067)

**Comment by pablomancera** [comment]
Compiling with and without the notmuch option did not make any changes to the generated manpage, and building the manpage seemed to be difficult for cross compiling and local compiling at least on an aarch64 device (due some issues compiling the pandoc dependency). So I think it's best to bundle the generated manpage as a file and install it with the template.

With this change the generated package worked on a Raspberry Pi 4, both cross compiling from my PC and compiling locally.

File: srcpkgs/i3status-rust/files/i3status-rs.1
Comment (classabbyamp) [inline srcpkgs/i3status-rust/files/i3status-rs.1:1]: no we are not going to vendor a 7.5k line man page from upstream

**Review by classabbyamp — CHANGES_REQUESTED** [review]

File: srcpkgs/i3status-rust/files/i3status-rs.1
Comment (kartikynwa) [reply srcpkgs/i3status-rust/files/i3status-rs.1:1]: For conditionally generating man pages you can look at this for reference: https://github.com/void-linux/void-packages/pull/44917/files

**Comment by pablomancera** [comment]
Superseded by #46079

**Pull request closed without merging**